test/d2l-fetch-auth/d2l-fetch-auth-framed.test.js

```diff
@@ -22,7 +24,7 @@ describe('d2l-fetch-auth', () => {
 	});
 
 	it('should create the d2lfetch object if it doesn\'t exist', () => {
-		expect(window.d2lfetch).to.be.defined;
+		expect(window.d2lfetch).to.not.be.undefined;
```

Comment:
Fun fact: this was passing before because `to.be.defined` isn't a thing in Chai and is itself `undefined`. But `undefined` matched `undefined` so it passed.

I think what was intended here was to import from the root directory and not from the `es6` directory. That way `d2lfetch` will get set.